tests/Fixer/Whitespace/StatementIndentationFixerTest.php

```diff
@@ -1072,6 +1072,148 @@ public function foo() {
 }',
             ['stick_comment_to_next_continuous_control_statement' => true],
         ];
+
+        yield 'if-elseif-else without braces' => [
+            '<?php
+if ($foo)
+    foo();
+elseif ($bar)
+    bar();
+else
+    baz();',
+            '<?php
+if ($foo)
+foo();
+elseif ($bar)
+  bar();
+else
+        baz();',
+        ];
+
+        yield 'for without braces' => [
+            '<?php
+for (;;)
+    foo();',
+            '<?php
+for (;;)
+  foo();',
+        ];
+
+        yield 'foreach without braces' => [
+            '<?php
+foreach ($foo as $bar)
+    foo();',
+            '<?php
+foreach ($foo as $bar)
+  foo();',
+        ];
+
+        yield 'while without braces' => [
+            '<?php
+while (true)
+    foo();',
+            '<?php
+while (true)
+  foo();',
+        ];
+
+        yield 'do-while without braces' => [
+            '<?php
+do
+    foo();
+while (true);',
+            '<?php
+do
+  foo();
+while (true);',
```

Comment:
```suggestion
do
  foo();
 while (true);',
```

Will `while` be un-intended? In general, I see the focus on statements "within" control structure, but maybe we can also make sure that edges of those structures are aligned?

Reply:
Test case updated